models/src/main/java/com/vimeo/networking2/Album.kt

```diff
@@ -0,0 +1,105 @@
+package com.vimeo.networking2
+
+import com.vimeo.networking2.enums.AlbumLayoutType
+import com.vimeo.networking2.enums.AlbumThemeType
+import com.vimeo.networking2.enums.SortType
+import java.util.*
+
+/**
+ * Group of videos to share publicly or privately.
+ */
+data class Album(
+
+    /**
+     * Hexadecimal color code for the decorative color.
+     * For example, album videos use this color for player buttons.
+     */
+    val brandColor: String? = null,
+
```

Comment:
There is also a custom_logo object that contains the following: uri, active, type, sizes (array), and resource_key.